styles/base.css

```diff
@@ -1,3 +1,7 @@
+* {
+  border: 1px solid blue;
```

Comment:
hahahahahaha, I will in no way keep this. It is/was for seeing where my different sections were for styling css.